mpas_analysis/shared/io/namelist_streams_interface.py

```diff
@@ -37,7 +37,7 @@ def convert_namelist_to_dict(fname, readonly=True):
             match = regex.findall(line)
             if len(match) > 0:
                 # assumes that there is only one match per line
-                nml[match[0][0]] = match[0][1]
+                nml[string.lower(match[0][0])] = match[0][1]
```

Comment:
@akturner, this is fine but I think it would work equally well to do `match[0][0].lower()`, assuming `match[0][0]` is a `str`.  Would save you having to import `string`...